command/commands.go

```diff
@@ -129,50 +129,106 @@ const (
 	flagNameDelegatedAuthAccessors = "delegated-auth-accessors"
 )
 
-var (
-	physicalBackends = map[string]physical.Factory{
-		"inmem_ha":               physInmem.NewInmemHA,
-		"inmem_transactional_ha": physInmem.NewTransactionalInmemHA,
-		"inmem_transactional":    physInmem.NewTransactionalInmem,
-		"inmem":                  physInmem.NewInmem,
-		"raft":                   physRaft.NewRaftBackend,
-	}
+type vaultHandlers struct {
```

Comment:
Could this have a go doc please?

Reply:
Added in [c293ac0](https://github.com/hashicorp/vault/pull/27515/commits/c293ac0eb382666c16170377890ab8d37e8311af).